specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/storageaccounts.json

```diff
@@ -118,24 +123,80 @@
             "$ref": "storage.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "storage.json#/parameters/ResourceGroupNameParameter"
+            "$ref": "storage.json#/parameters/LocationParameter"
           },
           {
-            "$ref": "storage.json#/parameters/FarmIdParameter"
+            "$ref": "#/parameters/AccountIdParameter"
           },
           {
-            "$ref": "storageaccounts.json#/parameters/AccountIdParameter"
+            "$ref": "storage.json#/parameters/ApiVersionParameter"
           },
           {
-            "$ref": "storage.json#/parameters/ApiVersionParameter"
+            "$ref": "#/parameters/NewAccountNameParameter"
```

Comment:
Assuming I am reading this correctly.

You should mention in the description that this will undelete a storage account with a new storage account name if this parameter is provided.

Reply:
yes, I'll revise the description.